crypto/unstable_keystack.ts

```diff
@@ -144,12 +144,12 @@ export class KeyStack {
    * key can be found, the method returns `-1`.
    */
   async indexOf(data: Data, digest: string): Promise<number> {
-    for (let i = 0; i < this.#keys.length; i++) {
-      const cryptoKey = await this.#toCryptoKey(this.#keys[i]);
+    for (const key of this.#keys) {
+      const cryptoKey = await this.#toCryptoKey(key);
       if (
         await compare(digest, encodeBase64Url(await sign(data, cryptoKey)))
       ) {
-        return i;
+        return this.#keys.findIndex((item) => item === key);
```

Comment:
This feels like a perf regression. Can you add `!` in line 148 instead?

Reply:
Updated.